main.cpp

```diff
@@ -0,0 +1,27 @@
+#include "Logger.h"
+
+// minimal example
+int foo()
+{
+	ENABLE_FUNCTION_TRACING
+	Logger::LogTrace("Tracing callable function...");
+	Logger::LogDebug("Debugging!");
+	return 10;
+}
+
+int main()
+{
+	ENABLE_FUNCTION_TRACING
```

Comment:
I would like to see an example of how this logger can be used across multiple threads.